frontend/actions/src/main/java/com/virtuslab/gitmachete/frontend/actions/base/BaseCheckoutAction.java

```diff
@@ -10,6 +10,8 @@
 import java.util.Collections;
 import java.util.stream.Stream;
 
+import javax.annotation.Untainted;
```

Comment:
Use checkerframework annotations. I'm not sure if Taintedness Checker recognizes these ones (Nullness Checker surely recognizes a lot of `@Nullable`/`@NonNull` annotations from other libraries)